chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -57,17 +67,16 @@ def eval_semantic_segmentation(pred_label, gt_label, n_class):
     <https://arxiv.org/abs/1704.06857>`_. arXiv 2017.
 
     Args:
-        pred_label (array): An integer array of image containing
-            class labels as values, which is obtained from inference.
-            This has shape :math:`(N, 1, H, W)` or :math:`(1, H, W)`,
-            where :math:`N` is size of the batch, :math:`H` is the height
-            and :math:`W` is the width.
-        gt_label (array): An integer array of image containing
-            the ground truth class labels as values. A pixel with value
-            "-1" will be ignored during evaluation. Its shape is similar
-            to :obj:`pred_label`.
-            Its image size is equal to that of :obj:`pred_label`.
-            This should be a one channel CHW formatted image.
+        pred_label (iterable of arrays): A collection of predicted labels.
```

Comment:
`(iterable of arrays)` -> `(iterable of arrays or array)`
If this function is mainly called with an iterable of arrays, `pred_labels` is better.